getting_started/index.adoc

```diff
@@ -8,26 +8,56 @@
 :toc: macro
 :toc-title:
 
+[[introduction]]
+
+== Introduction
+
+{product-title} is a platform as a service
+(PaaS) offering from Red Hat that brings together Docker and Kubernetes, and
+provides and API to manage these services. {product-title} allows you to create
+and manage containers. Containers are standalone processes that run within their
+own environment.
```

Comment:
This sentence is repeated on the next line (l24). Maybe get rid of this one? The next has a little more context.